CMakeLists.txt

```
@@ -130,52 +133,36 @@ if(ASPECT_WITH_WORLD_BUILDER)
     SET(WORLD_BUILDER_SOURCE_DIR "${CMAKE_SOURCE_DIR}/contrib/world_builder/" CACHE PATH "" FORCE)
   ENDIF()
 
-  # Always include the header files.
-  INCLUDE_DIRECTORIES("${WORLD_BUILDER_SOURCE_DIR}/include/")
+  string(TOLOWER ${CMAKE_BUILD_TYPE} BUILD_TYPE_LOWER)
```

That last argument here looks like it's a built-in cmake something because you spell it all upper-case. But it's really just the name of a local variable.

There is an informal cmake convention that temporary variables are spelled as `_temp_variable`. So I would suggest using
```suggestion
  STRING(TOLOWER ${CMAKE_BUILD_TYPE} _build_type_lower)
```